src/SFA.DAS.FindEpao.Api/Models/GetCourseEpaoResponse.cs

```diff
@@ -12,5 +12,6 @@ public class GetCourseEpaoResponse
         public IEnumerable<EpaoDeliveryArea> EpaoDeliveryAreas { get; set; }
         public IEnumerable<GetDeliveryAreaListItem> DeliveryAreas { get; set; }
         public IEnumerable<GetCourseListItem> AllCourses { get; set; }
+        public List<GetCourseEpaoStandardsListItem> standardVersions { get; set; }
```

Comment:
I think this can be removed based on the Spec

Reply:
removed

Reply:
This is required as per spec 
![image](https://user-images.githubusercontent.com/77051469/143063621-a5f2067c-4575-428e-8289-8b286c9e8274.png)